pkg/controllers/disruption/helpers.go

```diff
@@ -282,6 +225,64 @@ func BuildNodePoolMap(ctx context.Context, kubeClient client.Client, cloudProvid
 	return nodePoolMap, nodePoolToInstanceTypesMap, nil
 }
 
+// BuildDisruptionBudgets prepares our disruption budget mapping. The disruption budget maps for each disruption reason the number of allowed disruptions for each node pool.
```

Comment:
The language was a bit confusing to me
```suggestion
// BuildDisruptionBudgets prepares our disruption budget mapping. The disruption budget maps each disruption reason to the number of allowed disruptions.
```